integration_tests/run-tests.sh

```diff
@@ -0,0 +1,12 @@
+set -e
+
+for test in $(dirname $0)/test_*.sh ; do 
+  test_name=$(basename $test)
+  echo ; echo
```

Comment:
In an effort to make the output look nice, this script looks horrible lol

Reply:
I thought it'd be better to keep this separate from the `full-integration-test.sh` file but maybe they should all go in one and just be called `run-integration-tests.sh`?

Reply:
Are these just to add newlines? Could just `\n` as part of the echo statement on line 6, but neither are beautiful, so no strong preference either way.

Reply:
I tried that initially (putting them in the string and outside the string) and that resulted in `nn ---------------->   Running test_10_states.sh <---------------- nn` :sigh: